src/api/spec/models/requests_data_table_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.describe RequestsDataTable do
+    let!(:user) { create(:confirmed_user, login: 'moi') }
+    let(:source_project) { create(:project_with_package) }
+    let(:source_package) { source_project.packages.first }
+    let(:target_project) { create(:project_with_package) }
+    let(:target_package) { target_project.packages.first }
+    let!(:request_one) {
+      create(:bs_request_with_submit_action,
+             creator: user,
+             priority: 'critical',
+             source_project: source_project,
+             source_package: source_package,
+             target_project: target_project,
+             target_package: target_package)
+    }
+    let!(:request_two) {
+      create(:bs_request_with_submit_action,
+             created_at: 2.days.ago,
+             creator: user,
+             source_project: source_project,
+             source_package: source_package,
+             target_project: target_project,
+             target_package: target_package)
+    }
+
+    context 'sort direction' do
```

Comment:
Can you please have a look at the [betterspecs guidelines](http://betterspecs.org/#contexts) for rspec because this isn't the way context is meant to be used.

Here is an example of how the spec could be written to follow the rspec guidelines (and also I've added tests for the untested public methods on this class - `draw`, `records_total`, `count_requests`:

https://gist.github.com/evanrolfe/eb422ca6766cd063b1f90012e9a97bf9